services/todo-legacy/controllers/todo-controller.ts

```diff
@@ -1,15 +1,17 @@
-import { api, Application, inject } from '@loopback/core';
+import { Application, inject } from '@loopback/core';
+import { api } from '@loopback/rest';
```

Comment:
@b-admike we export `@api` from `@loopback/rest` only for backward compatibility, it is now defined and exported from `@loopback/openapi-v2`

Not a necessary to change them to `@loopback/openapi-v2` in this PR since that package will be renamed in v3 in the near future.

Reply:
@jannyHou sounds good. Are you ok with making those changes in #78?

Reply:
Sure~ 👍